### ISSUE TITLE
Clearer set-up instructions?

### ISSUE THREAD

**Issue by ptdev** [stitchfix/resque-brain#39]
Hi there, I'm trying to setup this interface to manage our queues. But we're a php shop so don't really have any experience with Rails or Ruby so most of the terms and config options mean almost nothing to us.
We're using php-resque, a port of the original that maintains full redis compatibility with the original so any user interfaces should still work.

Our issue is with actually making this project run since we're not RoR guys your "Deploy Resque Brain" explanation on the "Set-Up" wiki page is pretty much useless for any non Rails people.

That being said, we've been researching and reading about rails on the web so we can get this running and this is what we've done so far and where we are at:

- We cloned the project and ran "bundle install".
- We then generated a secret by running "rake secret" and adding the generated key into "config/secrets.yml"
- We're starting the server with: "RAILS_ENV=production rails server"

We then setup a proxy on our nginx so we can access the server through: "http://our_ip/resque"

Now, when accessing it, the system asks for http auth, but we cannot access it because we're not sure how to set these parameters. 
I see some HTTP_AUTH variables in ".env.production" and tried those but still cannot access it.

Can we get some feedback/support on how to get this up and running?

Is there anyway to have some clearer installation instructions for people that aren't RoR devs and just want to run an interface for their resque without having to dive into all the new Ruby/RoR terminology and different alternatives for running ruby/rails projects?

Thanks in advance. Best regards.

**Comment by davetron5000** [comment]
Hey, I appreciate your frustration.  I agree there's a lot left unsaid if you aren't a Rails developer.  A couple things:

* Do you have the ability to set environment variables for the app?  If so, you have to set two environment variables (see https://github.com/stitchfix/resque-brain/wiki/Set-Up#configure-http-auth ) for the http auth name/password.  Doing that and restarting the app, you should be able to access it using the configured name and password.
* You'll also need to set environment variables for the resque instance(s) to monitor.  Hopefully the [example](https://github.com/stitchfix/resque-brain/wiki/Set-Up#example) is clear as to how that works, but let me know if not

Now, assuming you do all this and it still doesn't work: this is where it's hard for me to help as it depends on your specific set up.  In the past, I've used [Phusion Passenger](https://www.phusionpassenger.com/) as the way to run Rails apps on a UNIX box.  I have very little sysadmin skills and was able to get it going on a Linode box pretty easily.  The thing to remember is to monitor all the various logs: the web server's, passenger's, and the Rails apps.

Anyway, let me know if this helps and if you get it working.  Based on what you end up discovering, we can update the wiki page for future people.

**Comment by ptdev** [comment]
Hi, thanks for the reply.

I ended up just running webrick with "rails server -b 0.0.0.0" and accessing it directly on port 3000 and in dev mode. I've setup my resques on the existing .env.development file.

We tinkered a bit with it and also with unicorn and nginx i couldn't make it run in production mode and also couldn't create the nginx proxy as we wanted.
We'll probably revisit this configuration farther down the line but for now unfortunately we can't lose too much time on this, we just needed some basic monitoring.

But anyway, it seems to be running and monitoring our resques just fine for now. I'll try and remember to post an update if/when we manage to set it up in an easier way for non-rails devs.

The UI is very cool, so good job with it! And thanks again.
